license.md

```diff
@@ -1,6 +1,6 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2017 Zeit, Inc.
+Copyright (c) 2021 Vercel, Inc.
```

Comment:
I think this should be 2017-2021 but not sure, probably need to ask legal

Reply:
I copied the existing repos (they all use latest version)